clang-tools-extra/test/clang-tidy/checkers/bugprone/Inputs/system-header-simulator-cxx.h

```diff
@@ -0,0 +1,1450 @@
+// Like the compiler, the static analyzer treats some functions differently if
```

Comment:
Quick nit: this comment refers to the "static analyzer", you should probably change it to a reference to "Clang Tidy".

Reply:
Will do, thanks Donat